neuralpp/symbolic/normalizer.py

```diff
@@ -38,7 +39,12 @@ class Normalizer:
     simplifier = ContextSimplifier()
 
     @staticmethod
-    def _normalize(expression: Expression, context: Z3SolverExpression) -> Expression:
+    def _basic_normalize(expression: Expression, context: Z3SolverExpression) -> Expression:
```

Comment:
renamed this `_normalize` to `_basic_normalize`, since we now have a `_quantifier_expression_normalize`